wails-frontend/src/layouts/DefaultLayout/DefaultLayout.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+interface LayoutProps {
+  children: ReactNode;
+}
+
+export default function DefaultLayout(props: LayoutProps) {
+  return <div className={`bg-primary min-h-screen`}>{props.children}</div>;
```

Comment:
I think we should make a separate pr for the default layout and create and refactor the wails screen to implement